Dockerfile_worker

```diff
@@ -37,31 +37,51 @@ MAINTAINER Jelmer Vernooij <jelmer@jelmer.uk>
 
 RUN apt-get update --yes \
  && apt-get install --yes --no-install-recommends \
-         uto-apt-proxy \
+        auto-apt-proxy \
         iproute2 \
  && apt-get upgrade --yes \
  && apt-get satisfy --yes --no-install-recommends \
-        autoconf \
```

Comment:
these dependencies are actually required to build packages and for sbuild to find missing dependencies